src/lib/form-builder/core/FieldRenderer.tsx

```
@@ -121,18 +135,27 @@ const FieldRendererComponent: React.FC<FieldRendererProps> = ({ field, value, on
-        error={error}
-        fieldErrors={fieldErrors}
-        fieldPath={fieldPath}
-        componentData={componentData}
-        formData={formData}
-        highlightedField={highlightedField}
-        highlightRequestId={highlightRequestId}
-    />;
+    // Wrap in a div to capture focus for non-translatable fields
+    return (
+        <div
+            role="group"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<fieldset>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)